tap_kafka/sync.py

```diff
@@ -227,6 +227,16 @@ def bookmarked_partition_to_next_position(topic: str,
         raise InvalidBookmarkException(f"Invalid bookmark. One or more bookmark entries using invalid type(s).")
 
 
+def seek_partitions(consumer, partitions):
+    """Seek partitions to offsets"""
+
+    LOGGER.info(f"Seek Partitions to Offsets {partitions}")
+    for partition in partitions:
+        consumer.seek(partition)
+
+    return consumer
```

Comment:
since `seek` method updates the consume offset, there is no need to return `consumer`
in the other word. consumer is an input parameter which is manipulated inside the method and no need to return it.